src/main/java/teammates/logic/automated/AdminEmailWorkerServlet.java

```diff
@@ -53,20 +49,19 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) {
         
         try {
             sendAdminEmail(emailContent, emailSubject, receiverEmail);
-            log.info("email sent to " + receiverEmail);
-        } catch (MessagingException | JSONException | IOException e) {
-            log.severe("Unexpected error while sending admin emails " + e.getMessage());
+            log.info("Email sent to " + receiverEmail);
+        } catch (Exception e) {
```

Comment:
Yeah got it. This was probably the reason why some emails failed to get sent without us realising?